aptos-move/aptos-transactional-test-harness/tests/aptos_test_harness/borrow_in_loop.exp

```diff
@@ -1,7 +1,7 @@
 processed 4 tasks
 
 task 1 'publish'. lines 4-26:
-Error: Failed to execute transaction. ExecutionStatus: MiscellaneousError(Some(STLOC_UNSAFE_TO_DESTROY_ERROR))
+Error: VMError with status STLOC_UNSAFE_TO_DESTROY_ERROR at location Module ModuleId { address: f75daa73fc071f93593335eb9033da804777eb94491650dd3f095ce6f778acb6, name: Identifier("m") } at index 0 for function definition at code offset 11 in function definition 0
```

Comment:
what is this?

Reply:
Before, the code was failing in verifier and converted to `VMStatus`. I changed the code so that for a module, we run the verifier, and then add directly to the data store, so the error is not an execution status anymore but the raw `VMError` from the verifier